grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -4118,9 +4118,17 @@ def section_udf(outer_panels):
                             "udf_latency_p99 - {{%s}}" % NODE_LABEL,
                         ),
                         panels.target(
-                            f"sum(irate({metric('udf_latency_sum')}[$__rate_interval])) / sum(irate({metric('udf_latency_count')}[$__rate_interval])) by ({COMPONENT_LABEL}, {NODE_LABEL})",
+                            f"sum(irate({metric('udf_latency_sum')}[$__rate_interval])) by ({COMPONENT_LABEL}, {NODE_LABEL}) / sum(irate({metric('udf_latency_count')}[$__rate_interval])) by ({COMPONENT_LABEL}, {NODE_LABEL})",
                             "udf_latency_avg - {{%s}}" % NODE_LABEL,
                         ),
+                        panels.target(
+                            f"histogram_quantile(0.99, sum(irate({metric('udf_latency_bucket')}[$__rate_interval])) by (le, link, name))",
+                            "udf_latency_p99_by_name - {{link}} {{name}}",
+                        ),
```

Comment:
Shall we also add p50, p90

Reply:
Adding P50 and P90 make the grafana panel overly cluttered. I think P99 and avg are more critical so I only add them. If they are really required in production, we can easily add them by manually adding new charts in Grafana.